code/network/multiutil.cpp

```diff
@@ -210,13 +223,16 @@ void multi_set_network_signature( ushort signature, int what_kind )
 	} else if ( what_kind == MULTI_SIG_ASTEROID ) {
 		Assert( (signature >= ASTEROID_SIG_MIN) && (signature <= ASTEROID_SIG_MAX) );
 		Next_asteroid_signature = signature;
+	} else if ( what_kind == MULTI_SIG_WAYPOINT ) {
+		Assert( (signature >= WAYPOINT_SIG_MIN) && (signature <= WAYPOINT_SIG_MAX) );
```

Comment:
The signature isn't set here. Is that intentional?

Reply:
That is a big error I made!  Looks like that allows my test case of *one* waypoint to work but would fail in other cases.